src/TailwindBuilder.php

```diff
@@ -102,28 +106,43 @@ public function setOutput(SymfonyStyle $output): void
         $this->output = $output;
     }
 
-    public function getInternalOutputCssPath(): string
+    public function getInternalOutputCssPath(string $inputPath): string
     {
-        return $this->tailwindVarDir.'/tailwind.built.css';
+        $inputFileName = pathinfo($inputPath, \PATHINFO_FILENAME);
+
+        return "{$this->tailwindVarDir}/{$inputFileName}.built.css";
     }
 
-    public function getInputCssPath(): string
+    public function getInputCssPaths(): array
     {
-        return $this->inputPath;
+        return $this->inputPaths;
     }
 
     public function getConfigFilePath(): string
     {
         return $this->configPath;
     }
 
-    public function getOutputCssContent(): string
+    public function getOutputCssContent(?string $inputFile = null): string
```

Comment:
Is it really can be null at some point? As I see we always pass a value to it

Reply:
You're right, I'll fix this.
Also a leftover of the first implementation